src/main/java/ch/wisv/areafiftylan/extras/consumption/controller/ConsumptionController.java

```diff
@@ -47,6 +48,11 @@ public ConsumptionController(ConsumptionService consumptionService) {
         return consumptionService.getPossibleConsumptions();
     }
 
+    @GetMapping("/consumptionMaps")
+    public Collection<ConsumptionMap> getAllConsumptionMaps() {
+        return consumptionService.getConsumptionMaps();
```

Comment:
What sort of respone will this give? An array with a map containing all made consumptions? An example would be useful. Good word though :+1: